source/modulo_controllers/test/test_controller_interface.cpp

```diff
@@ -186,7 +188,23 @@ TYPED_TEST_P(ControllerInterfaceTest, OutputTest) {
   }
 }
 
-REGISTER_TYPED_TEST_CASE_P(ControllerInterfaceTest, ConfigureErrorTest, InputTest, OutputTest);
+TYPED_TEST_P(ControllerInterfaceTest, CustomOutputTest) {
```

Comment:
If the rest of the PR makes sense, I think it would be reasonable that we merge this test with the above function.

Reply:
If possible then yes

Reply:
I'll spend a quick moment to see if I can make the tests a bit nicer, but otherwise the ones I added already test enough.